tools/bouncer_worker/bouncer_worker.js

```diff
@@ -47,6 +47,11 @@
 	let retry = 0;
 	let connClosed = false;
 
+	const fs = require("fs");
+	const configFullPath = path.resolve(__dirname, "config.json");
+	const conf = JSON.parse(fs.readFileSync(configFullPath));
```

Comment:
This conflicts with earlier declaration of `conf` on Line 35.

Reply:
Shouldn't Line 35 be removed?

Reply:
> Shouldn't Line 35 be removed?
yes